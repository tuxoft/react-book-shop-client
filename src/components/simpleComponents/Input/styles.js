import styled from "styled-components";

export const Input = styled.input.attrs({ type: "text" })`
  height: var(--input-height);
  border-radius: var(--input-border-radius);
  border: 1px solid var(--input-border-color);
  padding: var(--input-padding);
  background-color: var(--input-background-color);
  background: var(--input-background);
  color: var(--input-color);
  font-size: var(--input-font-size);
  width: var(--input-width);

  ${props => props.small && `
    height: var(--input-height-small);
  `}
`;