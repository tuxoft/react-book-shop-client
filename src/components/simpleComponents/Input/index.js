import React from "react";
import * as styles from "./styles";

const Input = (props) => (
  <styles.Input {...props} />
);

export default Input;
