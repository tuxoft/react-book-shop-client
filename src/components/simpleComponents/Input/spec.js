import React from "react";
import Input from "./index";
import { shallow } from "enzyme";

it("input", () => {
  const input = shallow(<Input />);
  expect(input).toMatchSnapshot();
});
