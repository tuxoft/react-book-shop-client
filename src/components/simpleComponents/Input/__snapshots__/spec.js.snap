// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`input 1`] = `<styled.input />`;
